data/json/recipes/recipe_food.json

```diff
@@ -526,6 +527,61 @@
       [ [ "soysauce", 1 ], [ "horseradish", 1 ], [ "salt", 1 ], [ "seasoning_salt", 1 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "tofu",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_VEGGI",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "book_learn": [ [ "cookbook_sushi", 3 ] ],
+    "time": 45000,
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ], 
+    "charges": 4,
+    "tools": [ [ [ "rag", -1 ] ], [ [ "surface_heat", 10, "LIST" ] ] ],
+    "batch_time_factors": [ 50, 2 ],
+    "components": [ [ [ "soy_milk", 5 ] ], [ [ "water_clean", 1 ] ], [ [ "chem_acetic_acid", 1 ] ] ]
```

Comment:
Might be a much better idea to use vinegar (diluted acetic acid, basically) here. Concetrated acetic acid is corrosive to the skin, so eating that will likely be a bad idea.